src/version.h

```diff
@@ -24,15 +24,15 @@
 #define CLASS 1
 
 #ifndef MAJOR_VERSION
-#define MAJOR_VERSION 0
+#define MAJOR_VERSION 2
 #endif
 
 #ifndef MINOR_VERSION
-#define MINOR_VERSION 0
+#define MINOR_VERSION 5
 #endif
 
 #ifndef PATCH_VERSION
-#define PATCH_VERSION 0
+#define PATCH_VERSION 1
 #endif
```

Comment:
These values are already set in the Makefile.
I'm not sure updating them is the right things to do.
Letting them to `0` tell us that something is wrong if the values are not defined.
Perhaps, we can just remove this code ? (Can you try this and check what happens if these values are not defined)

Reply:
Removed the values and all tests pass.